### ISSUE TITLE
Fullscreen image is not crip as original one

### ISSUE THREAD

**Issue by icaraps** [headwirecom/helix-sportsmagazine#103]
For example on https://main--helix-sportsmagazine--headwirecom.hlx.page/content-v2/golf-news-tours/_default/article/2023/7/3m-open-lee-hodges-wins-big-lead-sets-up-fedex-cup-playoffs which was imported from https://www.golfdigest.com/story/3m-open-lee-hodges-wins-big-lead-sets-up-fedex-cup-playoffs, you can tell that the hero image is too compressed which results in a blurry image.

**Comment by icaraps** [comment]
This wil require playing with the fastly image api which is used in franklin for image optimization https://developer.fastly.com/reference/io/ to find the best balance between image quality and performance.

**Comment by icaraps** [comment]
@hsaginor we found out that the imported image in this case here is of low quality in the document and therefore we can't display a better quality. Do you think this is something that can be fixed in the importer ? Would it be possible to retrieve the best image quality ?

**Comment by hsaginor** [comment]
@icaraps Please check if the attached document is any better. I am now trying to only import original renditions. I don't think we can do any better than the original.
 
[3m-open-lee-hodges-wins-big-lead-sets-up-fedex-cup-playoffs.docx](https://github.com/headwirecom/helix-sportsmagazine/files/12600165/3m-open-lee-hodges-wins-big-lead-sets-up-fedex-cup-playoffs.docx)

**Comment by icaraps** [comment]
> I don't think we can do any better than the original.

Yes original is better. There's anyway a limit in google docs. It will be downsized by default to 1600x1600 if the image is bigger than that.
